tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -362,13 +362,13 @@ def test_test_multiple_code_mapping_matches_order(self):
         configs = self.code_mappings
         # Expected configs: stack_root, automatically_generated
         expected_config_order = [
-            self.code_mapping1,  # "", False
             self.code_mapping3,  # "usr/src/getsentry/", False
             self.code_mapping4,  # "usr/src/", False
-            self.code_mapping2,  # "usr/src/getsentry/src/", True
+            self.code_mapping1,  # "", False
             self.code_mapping5,  # "usr/src/getsentry/src/sentry/", True
+            self.code_mapping2,  # "usr/src/getsentry/src/", True
         ]
```

Comment:
Check that the code mappings are correctly ordered now that stack root precedence is added